cli/src/pcluster/config/cluster_config.py

```diff
@@ -1388,6 +1389,9 @@ def __init__(
         self.networking = networking
         self.ssh = ssh or HeadNodeSsh(implied=True)
         self.local_storage = local_storage or LocalStorage(implied=True)
+        self.internal_shared_storage_type = Resource.init_param(
```

Comment:
Shouldnt there be Validators on the string to match either EFS or EBS? And not Fsx for example